config/tls.ini

```diff
@@ -15,6 +15,12 @@
 ; requestCert=true
 ; requestOCSP=false
 
+; rejectUnauthorized above requires verified TLS certs on EVERY TLS connection. When
+; rejectUnauthorized=false (default), you can require verified TLS certs on only the
```

Comment:
Wouldn't it be more secure to have this default to true?

Reply:
Yes, and unplugging the network cable would also be more secure. Neither one is very useful for MTA operators wanting to receive emails.

Reply:
Typical connections that send mail to a MTA will not present a valid TLS client certificate. If you set `rejectUnauthorized=true`, then you'll block the vast majority of mail. This is really an "edge case" feature, where you have clients that will authenticate to Haraka with a signed certificate instead of user/pass.

Reply:
Yeah, I shot a bit quickly and was only thinking about using Haraka as an outbound mail server. In this case it would only send emails to other MTAs that support TLS with this option, no?

Reply:
Likely the best way to make this point is to refer you to your Haraka mail logs with grep. Here's a few of my connections:

````
# grep verified=false /var/log/maillog | wc -l             
    1165
# grep verified=true /var/log/maillog | wc -l
      11
````

Reply:
> In this case it would only send emails to other MTAs that support TLS with this option, no?

Not quite. Haraka outbound would only be able to send mail to remote MTAs that present valid (is, verified by a CA that's included in the CAs that are bundled with Node.js) TLS certificates. This would be a higher than for inbound, but I think you'd still end up with lots of undelivered mail in your outbound queue.